setup.py

```diff
@@ -68,13 +68,16 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['hyperspy>=1.4.dev', 'ipywidgets>=6.0', 'link_traits'],
+    install_requires=['hyperspy>=1.5.dev', 'ipywidgets>=6.0', 'link_traits'],
```

Comment:
I am not sure, if this is good to keep `hyperspy>=1.5.dev`, I would simply put `hyperspy>=1.5` to avoid keeping the `.dev` in the release version.

Reply:
I suggest to leave it like this until v1.5 is released, then we'll release the gui packages and remove the dev from there. Otherwise installing this won't work until we release v1.5.

Reply:
Ah yes, that's right, 1.5 is newer than 1.5.dev...